### PR TITLE
APPSRE-3341: Fix owned_saas_files json schema definition

### PR DESCRIPTION
**Pull request by lechuk47**
Fix schema for `owned_saas_files` in `role-1` definition

### CODE REVIEW

**Review by maorfr — APPROVED** [review]
AMAZING

File: schemas/access/role-1.yml [hunk 1]
```diff
@@ -101,11 +101,15 @@ properties:
...
-        "$schemaRef": "/app-sre/saas-file-2.yml"
+      "$ref": "/common-1.json#/definitions/crossref"
+      "$schemaRef":
+        type: object
+        properties:
+          '$schema':
+            type: string
+            enum:
+              - "/app-sre/saas-file-1.yml"
+              - "/app-sre/saas-file-2.yml"
```
Comment (rporres) [inline schemas/access/role-1.yml:112]: nit: The general style of the document is to avoid indenting arrays.

**Review by rporres — APPROVED** [review]
Nice solution.

Just to add steps into solving our general lack of description in the schema, if you could just add a small `description` into this field you're fixing, it would be great.

File: schemas/access/role-1.yml (hunk 1 above)
Comment (lechuk47) [reply schemas/access/role-1.yml]: changed.

**Pull request merged**